packages/java/src/components/Class.tsx

```diff
@@ -0,0 +1,33 @@
+import { Child, Children, code, mapJoin, Scope } from "@alloy-js/core";
+import { Declaration, DeclarationProps } from "./Declaration.js";
+import { useJavaNamePolicy } from "../name-policy.js";
+import { AccessModifier, collectAccessModifier, collectModifiers, ObjectModifiers } from "../object-modifiers.js";
+import { Name } from "./Name.js";
+import { collectArguments } from "../arguments.js";
+
+export interface ClassProps extends DeclarationProps, ObjectModifiers {
+  accessModifier?: AccessModifier;
+  extends?: Children;
+  implements?: Children;
+}
+
+export function Class(props: ClassProps) {
+  if (Array.isArray(props.extends) && props.extends.length > 1) {
```

Comment:
This check isn't good because an array of children doesn't necessarily mean that it represents multiple extends, it could just be an array of strings (e.g. something like `["foo", "<", "T", ">" ]`)